web/src/main/java/org/akaza/openclinica/controller/DicomController.java

```diff
@@ -0,0 +1,58 @@
+package org.akaza.openclinica.controller;
+
+import org.akaza.openclinica.bean.login.UserAccountBean;
+import org.akaza.openclinica.service.DicomServiceClient;
+import org.akaza.openclinica.service.UtilService;
+import org.akaza.openclinica.service.randomize.RandomizationService;
+import org.akaza.openclinica.web.util.ErrorConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Controller to handle DICOM upload.
+ * @author svadla@openclinica.com
+ */
+@Controller
```

Comment:
Recommendation: use @RestController

@RestController is a stereotype annotation that combines @ResponseBody and @Controller. More than that, it gives more meaning to your Controller and also may carry additional semantics in future releases of the framework.